vector/src/main/java/im/vector/app/features/home/room/detail/composer/MessageComposerViewModel.kt

```diff
@@ -151,7 +150,7 @@ class MessageComposerViewModel @AssistedInject constructor(
 
     private fun handleOnTextChanged(action: MessageComposerAction.OnTextChanged) {
         val needsSendButtonVisibilityUpdate = currentComposerText.isBlank() != action.text.isBlank()
-        currentComposerText = SpannableString(action.text)
+        currentComposerText = action.text
```

Comment:
@bmarty Why are u doing that? 
As it uses the CharSequence from the editor, it'll break the `needsSendButtonVisibilityUpdate` as `currentComposerText` will always be equals to `action.text`.
Reverting the change still allow to keep formatting when switching mode as far as I see.

Reply:
You're right, it is a mistake to change this. Reverted in 2395d72c93da67871c5a2d4ed61d8f93978481c0